### ISSUE TITLE
For tensorflow cpu version

### ISSUE THREAD

**Issue by mzuhairqadir** [markjay4k/Mask-RCNN-series#2]
If you wish to run this demo without GPU then instead of renaming GPU in the flowing line

tensorflow-cpu==1.5

you just use

tensorflow and it will by default install CPU version and i test run with latest 1.6 version of tensorflow CPU version. No problem there.

**Comment by markjay4k** [comment]
thats correct

**Comment by cooleel** [comment]
What is the keras version you are using with tensorflow 1.5? 
I met some error when trying to run the inference mode on the CPU.
The same script runs well on a GPU machine.
I wonder if the versions of tensorflow and keras matter.
Thanks.

**Comment by bhuiyanmobasshir94** [comment]
@mzuhairqadir I am facing problem working with tensorflow CPU. Can you help, please?
